gallery/src/components/NxDropdown/NxDropdownNavigationExample.tsx

```diff
@@ -15,14 +15,14 @@ function NxDropdownNavigationExample() {
 
   return (
     <NxDropdown label="Navigation" isOpen={isOpen} onToggleCollapse={onToggleCollapse}>
-      <a onClick={onClick} className="nx-dropdown-button">
-        Nav Link1
+      <a onClick={onClick} href="#/pages/NxDropdown" className="nx-dropdown-button">
+        Text link 1
       </a>
-      <a onClick={onClick} className="nx-dropdown-button">
-        Nav Link2
+      <a onClick={onClick} href="#/pages/NxDropdown" className="nx-dropdown-button">
+        Text link 2
       </a>
-      <a onClick={onClick} className="nx-dropdown-button">
-        Nav Link3
+      <a onClick={onClick} href="#/pages/NxDropdown" className="nx-dropdown-button">
+        Text link 3
```

Comment:
If you're going to change these to make them less confusing, you should probably also change the buttons ones below.  Having them referred to have "Nav Links" doesn't really make sense.

Reply:
> If you're going to change these to make them less confusing, you should probably also change the buttons ones below. Having them referred to have "Nav Links" doesn't really make sense.

Made them consistent across all examples: https://github.com/sonatype/sonatype-react-shared-components/pull/212/commits/426c00100d3915785033dcd7766b8a6191beea43